qiskit/qpy/type_keys.py

```diff
@@ -231,10 +232,11 @@ class ScheduleInstruction(TypeKeyBase):
     SET_PHASE = b"q"
     SHIFT_PHASE = b"r"
     BARRIER = b"b"
+    TIME_BLOCKADE = b"t"
```

Comment:
Do we need to modify the format documentation to mention this type key in the format specification for qpy now?

Reply:
Good catch. Done in https://github.com/Qiskit/qiskit-terra/pull/8949/commits/1756e35a9d7ed57101a934ccf0eb6148ba57a31c. Since I didn't increment QPY version I updated the note of version 5.